atom/browser/browser_win.cc

```diff
@@ -125,15 +125,47 @@ bool Browser::SetUserTasks(const std::vector<UserTask>& tasks) {
   return SUCCEEDED(destinations->CommitList());
 }
 
-bool Browser::RemoveAsDefaultProtocolClient(const std::string& protocol) {
-  if (protocol.empty())
-    return false;
+std::wstring protocolLaunchPath(std::string protocol, mate::Arguments* args) {
+  // Read in optional exe path arg
+  std::wstring exePath;
+  std::string rawExePath;
```

Comment:
Should store the path in `wstring` or `FilePath`, otherwise unicode characters may fail to convert.